storybook/main.qml

```diff
@@ -20,18 +20,16 @@ ApplicationWindow {
 
     title: "%1 – %2".arg(currentPage).arg(Qt.application.displayName)
 
-    palette.window: Theme.palette.statusAppLayout.backgroundColor
-    palette.text: Theme.palette.directColor1
-    palette.windowText: Theme.palette.directColor1
-    palette.base: Theme.palette.indirectColor1
     font.pixelSize: 13
 
+    Universal.theme: darkModeCheckBox.checked ? Universal.Dark : Universal.Light
+
     onCurrentPageChanged: {
         testsReRunTimer.restart()
         settings.setValue("currentPage", currentPage)
     }
 
-    Component.onCompleted: currentPage = settings.value("currentPage")
+    Component.onCompleted: currentPage = settings.value("currentPage") ?? "Playground"
```

Comment:
Plese move it directly to the property declaration to avoid warning like
```
status-desktop/storybook/pages/Page.qml: No such file or directory
```
```qml
property string currentPage: settings.value("currentPage") ?? "Playground"
```